### ISSUE TITLE
not able to clone the environment due to some error by semantic-ui.

### ISSUE THREAD

**Issue by jaskeerat789** [coding-blocks/boss#185]
`npm install ` is giving error due to some failure caused by semantic-ui and gulp.
This is not allowing me to clone  the environment. 
node-10.15.3
npm-6.9.0
os-ubuntu-disco
![Screenshot from 2019-05-19 22-28-25](https://user-images.githubusercontent.com/38110126/57985432-9628b500-7a85-11e9-8b8b-4d7ea6991c43.png)

**Comment by TheOneAboveAllTitan** [comment]
I'm having same issue
node- v11.15.0
npm - 6.9.0  
OS - Linux Manjaro x86_64
npm logs - http://pasted.co/32abfa39

**Comment by TheOneAboveAllTitan** [comment]
Hey @jaskeerat789 
I was having same problem, I downgraded node version to 8 and now it works.

**Comment by mzfr** [comment]
Yes, this issue was caused after updating. This can be fixed by running `npm install natives@1.1.6`.

**Comment by mritunjaysharma394** [comment]
Even I was facing the same issue. Just downgrade the version and it works

**Comment by TheOneAboveAllTitan** [comment]
There is a fix for this in #189
